project-base/tests/FrontendApiBundle/Functional/Product/ProductsTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Functional\Product;
+
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Tests\FrontendApiBundle\Test\GraphQlTestCase;
+
+class ProductsTest extends GraphQlTestCase
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    private $domain;
+
+    protected function setUp(): void
+    {
+        $this->domain = $this->getContainer()->get(Domain::class);
+
+        parent::setUp();
+    }
+
+    public function testFirstFiveProductsWithName(): void
+    {
+        $firstDomainLocale = $this->domain->getDomainConfigById(Domain::FIRST_DOMAIN_ID)->getLocale();
+        $query = '
+            query {
+                products {
+                    name
+                }
+            }
+        ';
+
+        $productsExpected = [
+            ['name' => t('22" Sencor SLE 22F46DM4 HELLO KITTY', [], 'dataFixtures', $firstDomainLocale)],
+            ['name' => t('32" Philips 32PFL4308', [], 'dataFixtures', $firstDomainLocale)],
+            ['name' => t('47" LG 47LA790V (FHD)', [], 'dataFixtures', $firstDomainLocale)],
+            ['name' => t('A4tech mouse X-710BK, OSCAR Game, 2000DPI, black,', [], 'dataFixtures', $firstDomainLocale)],
+            ['name' => t('Apple iPhone 5S 64GB, gold', [], 'dataFixtures', $firstDomainLocale)],
+        ];
+
+        $queryResult = array_slice($this->getResponseContentForQuery($query)['data']['products'], 0, 5);
+
+        $this->assertEquals($productsExpected, $queryResult, json_encode($queryResult));
+    }
+
+    public function testSeventhProductWithAllAttributes(): void
+    {
```

Comment:
Apply same resolvers here as suggested in previous test

Reply:
Fixed https://github.com/shopsys/shopsys/pull/1471/commits/f8c97aed253d218f8aefc7bcad5a64855ed5b782